valdr-message/valdr-message.d.ts

```diff
@@ -1,4 +1,4 @@
-﻿// Type definitions for valdr.js v1.1.5 (valdr-message)
+﻿// Type definitions for valdr.js v1.1.5+ (valdr-message)
```

Comment:
this version number is used by other tools, e.g. publisher for @types. so please keep this to numbers.